pkg/account/account.go

```diff
@@ -0,0 +1,71 @@
+package account
+
+import (
+	"context"
+
+	"cosmossdk.io/math"
+	"go.uber.org/zap"
+
+	"github.com/archway-network/relayer_exporter/pkg/chain"
+	log "github.com/archway-network/relayer_exporter/pkg/logger"
+)
+
+type Account struct {
+	Address string `yaml:"address"`
+	Denom   string `yaml:"denom"`
+	ChainID string `yaml:"chainId"`
+	Balance math.Int
+}
+
+func GetBalances(accounts []Account, rpcs map[string]string) []Account {
+	num := len(accounts)
+
+	out := make(chan Account, num)
+	defer close(out)
+
+	for i := 0; i < num; i++ {
```

Comment:
I think waitGroups would be a simpler solution for this?
1. Start your goroutines in loop, assign them to WG
2. Wait for each goroutine to finish and continue.